### PR TITLE
simplified kcli plan

### PR DESCRIPTION
**Pull request by karmab**

### CODE REVIEW

File: testing/infra/machines.yml [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
+  path: /srv/
+
 vm_net:
  type: network
  cidr: 10.0.5.0/24
 
 micro_fedora:
   type: profile
-  image: Fedora-Cloud-Base-30-1.2.x86_64.qcow2
+  image: fedora30
```
Comment (cherusk) [inline testing/infra/machines.yml]: That convenience mechanism actually never worked for me! This long form has a reason.

Comment (cherusk) [reply testing/infra/machines.yml]: @karmab please, confirm. Not certain is it down to when the image storage sits on pools, but doubt.

File: testing/maskfile.md [hunk 2]
```diff
@@ -10,15 +10,8 @@ set -eEux
 
 echo "instanciating infra"
 
-kcli create host kvm -H 127.0.0.1 local
-
-kcli create plan -f "${SOFT_RESIDE}/infra/pools.yml" pools 
-kcli list pool
-kcli download image -p image_pool fedora30
-
```
Comment (cherusk) [inline testing/maskfile.md]: Also this was necessary, I iterated that out over several runs. I known it differs from the doc.

File: testing/maskfile.md [hunk 3]
```diff
@@ -10,15 +10,8 @@ set -eEux
...
 kcli create plan -f "${SOFT_RESIDE}/infra/machines.yml" micro_fedora
 kcli list image
-kcli -d start plan micro_fedora
```
Comment (cherusk) [inline testing/maskfile.md]: That's also not optional a step.

**Comment by cherusk** [comment]
As I have not continuous build on PRs yet, I'll merge in and revert, but then you can see the issues I described.

**Comment by cherusk** [comment]
Gosh, it worked well: https://travis-ci.com/cherusk/godon/jobs/288063189 :astonished:

**Comment by cherusk** [comment]
I intreat you, I saw these constellations falling. Anyhow.

File: testing/infra/machines.yml (hunk 1 above)
Comment (karmab) [reply testing/infra/machines.yml]: yeah, i fixed the convenience stuff some weeks ago

**Pull request merged**